tft/data_formatters/electricity.py

```diff
@@ -213,7 +213,7 @@ def format_predictions(self, predictions):
 
       for col in column_names:
         if col not in {'forecast_time', 'identifier'}:
-          sliced_copy[col] = target_scaler.inverse_transform(sliced_copy[col])
+          sliced_copy[col] = target_scaler.inverse_transform(sliced_copy[col].values.reshape(-1,1))
```

Comment:
Have you tested this trains without error?

Reply:
Yes, of cause.